test/functional/terminal/tui_spec.lua

```diff
@@ -85,6 +85,23 @@ describe('TUI', function()
     assert_alive()
   end)
 
+  it('works correctly at startup', function()
+    screen:try_resize(50, 10)
+    feed_command([[call termopen(['build/bin/nvim', '--clean', '--cmd', 'let start = reltime() | while v:true | if reltimefloat(reltime(start)) > 2 | break | endif | endwhile']) | sleep 500m | vs new]])
+    screen:expect([[
+      {1: }                        │                        |
+      {4:~                        }│{4:~                       }|
+      {4:~                        }│{4:~                       }|
+      {4:~                        }│{4:~                       }|
+      {4:~                        }│{4:~                       }|
+      {4:~                        }│{5:[No Name]   0,0-1    All}|
+      {4:~                        }│                        |
+      {5:new                       }{1:</neovim/build/bin/nvim }|
```

Comment:
can use `{MATCH:x}` so that we don't hardcode the path, see `screen.lua`.

```suggestion
      {5:new                       }{MATCH:<.*[/\]nvim }|
```